app/controllers/npq_separation/migration/migrations_controller.rb

```diff
@@ -8,8 +8,7 @@ def index
   end
 
   def create
-    Migration::Migrator.prepare_for_migration
-    MigrationJob.perform_later
+    ActiveJob.perform_all_later([MigrationJob.new, MigrationJob.new])
```

Comment:
Can we chat about this tomorrow, as Im not sure I understand, are we starting migrations in parallel?